docs/sources/getting-started/timeseries-dimensions.md

```diff
@@ -0,0 +1,85 @@
++++
+title = "Time series dimensions"
+description = "time series dimensions"
+keywords = ["grafana", "intro", "guide", "concepts", "timeseries", "labels"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/timeseries-dimensions"]
+[menu.docs]
+name = "Time series dimensions"
+identifier = "time_series_dimensions"
+parent = "guides"
+weight = 300
++++
+
+# Time series dimensions
+
+In at the end of the ["Time series databases" section of "Introduction to time series"]({{< relref "timeseries.md#time-series-databases" >}}) the concept of Labels (a.k.a tags) is introduced:
+
+> Another feature of a TSDB is the ability to filter measurements using _tags_. Each data point is labeled with a tag that adds context information, such as where the measurement was taken. ...
+
+With time series data, the data often contains more than a single series, and is a set of multiple time series. Many Grafana data sources support this type of data.
+
+{{< docs-imagebox img="/img/docs/example_graph_multi_dim.png" class="docs-image--no-shadow" max-width="850px" >}}
+
+The common case is issuing a single query for a measurement with one or more additional properties as dimensions. For example querying a temperature measurement along with a location property as a dimension of the measurement. In this case, multiple series are returned back from that single query. Each series in this case has unique location. To uniquely identify these series within a set of time series, Grafana stores this information in _Labels_.
+
+## Labels
+
+Each time series in Grafana optionally has Labels. Labels are set a of key/value pairs for identifying dimensions. Example Labels could are `{location=us}` or `{country=us,state=ma,city=boston}`. Within a set of time series, the combination of its name and labels identifies each series. For example, `temperature {country=us,state=ma,city=boston}`.
+
+Different sources of time series data have dimensions stored natively, or common storage patterns that allow the data to be extracted into dimensions. Time series databases (TSDBs) usually natively support dimensionality. In TSDBs such as Graphite or OpenTSDB the term _Tags_ is used - in Prometheus the same term as used as in Grafana - _Labels_. In table databases such SQL, these dimensions are generally the `GROUP BY` parameters of a query.
+
+By storing dimensionality as Labels for time series in Grafana it allows for the possibility of data transformations and alerting to also support multiple dimensions.
+
+## Multiple dimensions in table format
+
+In SQL or SQL-like databases that return table responses, additional dimensions are generally found as columns in the table that is returned as a response from the query.
+
+### Single dimension
+
+For example, consider a pseudo query like:
```

Comment:
I have never heard of a "pseudo query." Will this be understandable to your general audience?

Reply:
I meant Pseudocode SQL https://en.wikipedia.org/wiki/Pseudocode . But the point is just that it isn't any *actual* particular language  - so sort of "pretend" SQL.

But I think that point can be removed altogether -- I can just remove that word and put "consider a query like" given the scope/context of this document.

Reply:
Yes, I agree.